catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -106,6 +107,7 @@ public void run(CatalogApplicationConfig catalogConfig, Environment environment)
 
     // Register Event Handler
     registerEventFilter(catalogConfig, environment, jdbi);
+    EventPubSub.start();
```

Comment:
do we need to add .shutdown() in shutdown path to flush any messages pending?

Reply:
This is work in progress. Unfortunately, there is no Dropwizard Application shutdown to trigger this. Need to find a way to do this with Dropwizard. Will be handled as part #1901